app/controllers/admin/businesses/invisiblizes_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Admin
+  module Businesses
+    class InvisiblizesController < Admin::ApplicationController
```

Comment:
I don't think the `Admin::` namespace is needed here.

Reply:
Hm, why's that? do you mean you want me to move this out of the admin namespace?

Reply:
Oh! duh, by bad totally wasn't tracking. I am now 👍